src/main/java/com/sleekbyte/tailor/utils/CliArgumentParser.java

```diff
@@ -60,6 +60,15 @@
     private CommandLine cmd;
 
     /**
+     * Exception thrown when option parsing fails.
+     */
+    public static class CliArgumentParserException extends Exception {
```

Comment:
We can rename this exception and the surrounding class to `CLI` instead of `Cli`.